src/main/java/core/basesyntax/service/ReportService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.Map;
+
+public interface ReportService {
+    String generateReport(Map<String, Integer> fruitsMap);
```

Comment:
I think you should accept the database (fruitMap) as a parameter in the constructor of ReportService implementation